docs/extension/QUnit.extend.md

```diff
@@ -1,49 +0,0 @@
----
-layout: page-api
-title: QUnit.extend()
-excerpt: Copy the properties from one object into a target object.
-groups:
-  - extension
-  - deprecated
```

Comment:
I've prepared a `removed` group in the main branch and live at <https://api.qunitjs.com/removed/> where this can go. I'd prefer we keep the docs live to serve for past versions as well as people tend to use older versions for a long time, and we've been fairly good at making that stable and easy to do 🙂

Reply:
:+1:  done